src/Action/CheckForSpacesInEOL.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\GitHooks\Action;
+
+use CaptainHook\App\Config;
+use CaptainHook\App\Console\IO;
+use SebastianFeldmann\Git\Repository;
+use Symfony\Component\Process\Process;
+use function array_diff;
+use function implode;
+
+final class CheckForSpacesInEOL extends Action
```

Comment:
This should be named e.g. `CheckForWhiteSpaceAtEOL`, since it doesn't only remove spaces but all whitespace.

Reply:
The class is renamed now